app/selectors/permissions.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {createSelector} from 'reselect';
+
+import {Permissions} from '@mm-redux/constants';
+import {getCurrentUser} from '@mm-redux/selectors/entities/common';
+import {getMyChannelRoles, getMySystemPermissions, getMyTeamRoles, getRoles} from '@mm-redux/selectors/entities/roles';
+import {isPostOwner} from '@mm-redux/utils/post_utils';
+
+import type {Post} from '@mm-redux/types/posts';
+import type {GlobalState} from '@mm-redux/types/store';
+
+export const getMyTeamPermissions = createSelector(
+    getMyTeamRoles,
+    getRoles,
+    getMySystemPermissions,
+    (state: GlobalState, team_id: string) => team_id,
```

Comment:
Can this line be extracted to its own selector?

Reply:
For what purpose?